docker-compose.yml

```diff
@@ -1,18 +1,66 @@
-version: "3.9"
 services:
-  db:
-    image: postgres:14
+  app:
+    container_name: kelvin_app
+    depends_on:
+      - db
+      - redis
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      - DATABASE__HOSTNAME=db
+      - DATABASE__PORT=5432
+      - DATABASE__DB=${DATABASE__DB}
+      - DATABASE__USERNAME=${DATABASE__USERNAME}
+      - DATABASE__PASSWORD=${DATABASE__PASSWORD}
+      - REDIS__HOST=redis
+      - REDIS__PORT=6379
+    env_file:
+      - .env
+    restart: unless-stopped
     ports:
-      - "5432:5432"
-    volumes:
-      - ${KELVIN_DB_PATH?Absolute path to Kelvin database storage}:/var/lib/postgresql/data
+      - "${APP__PORT}:8000"
+
+  db:
     container_name: kelvin_db
+    image: postgres:14
     environment:
-      - POSTGRES_DB=${DB_DATABASE?Name of PostgreSQL database}
-      - POSTGRES_USER=${DB_USERNAME?PostgreSQL username}
-      - POSTGRES_PASSWORD=${DB_PASSWORD?PostgreSQL password}
-  redis:
-    image: redis
+      - POSTGRES_DB=${DATABASE__DB}
+      - POSTGRES_USER=${DATABASE__USERNAME}
+      - POSTGRES_PASSWORD=${DATABASE__PASSWORD}
+    env_file:
+      - .env
+    volumes:
+      - kelvin_db_data:/var/lib/postgresql/data
```

Comment:
Could you please keep the previous configurable path (`${KELVIN_DB_PATH?Absolute path to Kelvin database storage}:/var/lib/postgresql/data`) so that we can specify the name of the directory where the DB data will be stored? Thanks!